### DISCUSSION TITLE
Format Text & Text Color Accessibility

### DISCUSSION THREAD

**Discussion by Asstrole** [windingwind/zotero-better-notes#1195] [Ideas]
NOTE: This was originally opened in Action Script, but windingwind suggested:

>  It might be possible to do that with this plugin, and probably it would be better to add it to the Better Notes plugin's [Magic Key] 

which would work great. However, when selecting text and using the / key, it will overwrite the text. So, I am not sure how that will work for formatting text. 

Original Suggestion
I constantly use the Format Text and the Text Color drop down menus and would like some faster ways to use these. Does anyone have a script or could write one that could do any of the following:

- Make these menus (Format Text and Text Color) accessible via right click?
- Add keyboard shortcuts to the Menus (Format Text & Text Color) and/or menu items? (Text Color > Yellow Keyboard-Shortcut) ?
- Similar to "Show on Page and Unlink" popup that automatically appears when you click in quoted note text, could a similar popup be added or triggered via shortcut with the Format Text & Text Color options?
- or any other similar helpful way to access these without so much mouse work.

Thank you very much !

**Comment by windingwind** [comment]
Would adding a keyboard shortcut to open the popup be useful?

**Reply by Asstrole** [reply]
Yes that would be great! Tymuch

**Reply by Asstrole** [reply]
@windingwind Hey, I wanted to check in and see if adding color / highlighting to the menu (/) or a short-cut is still a consideration? Much gratitude!

**Reply by windingwind** [reply]
Yeah it’s planned, I just don’t have time for that right.

If you don’t mind, maybe we can create a new ticket for this request in a issue, so that I would not forget it

**Reply by shateiely** [reply]
This is a must-have feature for those who want to use the notes in depth!!!

**Reply by MHMengqi** [reply]
Same issue here. Has anyone found a workaround, perhaps using magic key or shortcut, to make highlighting text in notes faster? :)
